content/blog/2017-11-21-ochRe.md

```diff
@@ -22,8 +22,6 @@ tags:
   - unconf
 ---
 
-<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
there's i think 3 files where `https://platform.twitter.com/widgets.js` is removed, not sure what that does or if its needed still

Reply:
No longer needed, and I caught it whilst looking for embedded gists.

Reply:
Sorry for the lack of context